src/collection-facets.ts

```diff
@@ -88,9 +99,12 @@ export class CollectionFacets extends LitElement {
     year: false,
   };
 
+  @query('modal-manager') private modalManager!: any;
```

Comment:
you may need to drop the modal in using dependency injection as this is reaching global scope